client/components/post-card/search.jsx

```diff
@@ -53,7 +53,7 @@ export function SearchPostCard( { post, site, feed, onClick = noop, onCommentCli
 					commentCount={ post.discussion.comment_count }
 					tagName="span" showLabel={ false }
 					onClick={ onCommentClick }/>
-				<LikeButton siteId={ post.site_ID } postId={ post.ID } tagName="span" showCount={ true } showLabel={ false } />
+				<LikeButton siteId={ post.site_ID } postId={ post.ID } tagName="span" showCount={ false } showLabel={ false } />
```

Comment:
`showCount={ false }` implies that the count will never be shown, when we're actually showing it when > 0 - is that right?